src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/LogUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.utils;
+
+import org.apache.logging.log4j.ThreadContext;
+
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Utility class for generating/accessing the request id from logging context.
+ */
+public class LogUtils {
+
+    /**
+     * The key of the request id in the context map
+     */
+    public static final String REQUEST_ID_KEY = "request_id";
```

Comment:
`private` may be better? Since this is more like your internal detail rather than a public constant.

Reply:
Agree, no reason to keep it `public`. I was trying to do something fancier to basically check if the elastic's thread context has the id and get it from there, that's why made this `public`, but that would be a different change anyway (if we ever make it), changing this to `private`.